src/main/java/duke/command/TodoCommand.java

```diff
@@ -0,0 +1,46 @@
+package duke.command;
+
+import java.io.IOException;
+
+import duke.exceptions.DukeException;
+import duke.task.Task;
+import duke.task.Todo;
+import duke.util.DukeIo;
+import duke.util.ParsedData;
+import duke.util.Storage;
+import duke.util.TaskList;
+
+/**
+ * Command to add a todo task to the tasklist.
+ */
+public class TodoCommand extends DataCommand {
+
+    private static final String ADD_TASK = "Got it. I've added this task:%n"
+            + "  %s%n"
+            + "Now you have %d tasks in the list.";
```

Comment:
Perhaps the indentation of line 20 should be 8 spaces more than the parent line?

Reply:
Line 20's parent line should still be line 18, so I don't think there should another 8 spaces on line 20. 
I think the parent line should change when there is a change in scope. 
ie 
`demo()
        .method1()
        .method2(demo2        // <-- no change in parent line
                .method3());        // <-- change in parent line